app/src/main/java/com/nutomic/syncthingandroid/util/ConfigXml.java

```diff
@@ -62,8 +66,32 @@ public ConfigXml(Context context) throws OpenConfigException {
         readConfig();
 
         if (isFirstStart) {
-            changeLocalDeviceName();
-            changeDefaultFolder();
+            boolean changed = false;
+
+            /* Synthing devices */
+            changed = changeLocalDeviceName() || changed;
+
+            /* Syncthing folders */
+            changed = changeDefaultFolder() || changed;
+
+            /* Syncthing options */
+            /**
+              * As verifying the impact of disabling restartOnWakeup is not
+              * completed yet, we stick to what has been default in the past
+              * and keep this syncthing core feature enabled.
+              * After additional tests, we want to disable restartOnWakeup
+              * as advised by calmh, Nutomic, AudriusButkevicius to save battery.
+              * see https://github.com/syncthing/syncthing-android/issues/368
+              * and https://forum.syncthing.net/t/question-about-restartonwakeup-setting/2222/11
+              */
+            Element options = (Element) mConfig.getDocumentElement()
+                    .getElementsByTagName("options").item(0);
+            changed = setConfigElement(options, "restartOnWakeup", "true") || changed;
+
+            // Save changes if we made any.
+            if (changed) {
+                saveChanges();
+            }
```

Comment:
I don't think any change is needed here, as this sets it to the default value anyway.
Also, if we ever decide that the default should be the other way around, doing it under isFirstStart is not a good idea.

Reply:
Ok, took it out (committing soon)